<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsFL</class>
 <widget class="QWidget" name="QtWidgetsFL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件加载</string>
  </property>
  <widget class="QLabel" name="labelWind">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>站点及线路文件加载</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStop">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>站点文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLine">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>线路文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLine">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDone">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditStop">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelHits">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>地名文件</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditorg">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>176</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonorg">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonStop</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsFL</receiver>
   <slot>inputStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonQuit</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsFL</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLine</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsFL</receiver>
   <slot>inputLine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDone</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsFL</receiver>
   <slot>checkFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonorg</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsFL</receiver>
   <slot>inputOrg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputStop()</slot>
  <slot>inputLine()</slot>
  <slot>checkFile()</slot>
  <slot>inputOrg()</slot>
 </slots>
</ui>
